### ISSUE TITLE
Blocks with status Off continue to use power

### ISSUE THREAD

**Issue by TheQuadShot** [micdoodle8/Galacticraft#2845]
Galacticraft-Planets 1.10.2-4.0.0.107
GalacticraftCore 1.10.2.4.0.0.99
MiccdoodleCore 1.10.2-4.0.0.107

Galacticraft-Planets 1.11.2-4.0.0.106
GalacticraftCore 1.11.2-4.0.0.106
MiccdoodleCore 1.11.2-4.0.0.106

If a machine is off it should not consume power. I wanted to have an Emergency Oxygen Bubble Gen with battery and Lox canister to toggle with a lever, but its out of power after about 20 minutes. It consumes about 10gJ per second and it applies to all of the machines that use power.

**Comment by dj3520** [comment]
I believe redstone control feature was removed due to possible unintentional chunk loading. (though my branch has 1.7.10 versions with it re-enabled.)

**Comment by TheQuadShot** [comment]
If I toggle a machine that processes something it will stop processing and changes it's status. The exception is the Arc Furnace, it stops processing but it's status is still "Running"(a bug I was getting ready to file).

**Comment by radfast** [comment]
@TheQuadShot all the Galacticraft machines currently have a "feature" where they will lose a small amount of energy each second from their internal energy storage.  This is the ~10gJ/second you are seeing.

There are two main reasons for this:
- it seems realistic, there is no real world battery technology which can hold energy indefinitely, and especially re-chargeable batteries tend to lose energy noticeably over time
- when a machine has only a small amount of energy remaining, it fractionally improves the game performance if we reduce that to 0

This does have the consequence that even a machine which is "off" will lose all its energy over a long time (20 minutes is "a long time" in Minecraft).

We could very easily change this so that if the machine is disabled using a 'Disable' button or redstone control then it will not lose energy like this.  That loses realism, but is less annoying.  What do you think?

**Comment by TheQuadShot** [comment]
Maybe there should be a "Stand By" mode since "Off" leads me to believe that it will not consume power. I can see(and agree) using some power when idle(Stand By) on some machines like the arc furnace so that it stays at temperature so it can process ore quickly. However with things like the bubble gen, it does absolutely nothing when it's "off" and it doesn't need to be in "Stand By" since it is pretty much just a valve that opens when power is applied. I don't think there should be a disable option since I can use a redstone wire to "disable" it. It is also a bit inconsistent as far as the compressors are concerned. The compressor doesn't use/consume coal when it's not being used, but the electric compressor never stops using power. As an added note, I can't think of any "machine" from other mods that consume power when not in use and/or in an "off" condition anymore.

What I would like to do is have a stand alone (battery and Lox)Bubble Gen. or Ox Compressor that can be enabled via a redstone signal, lever or Ox Detector. These would be "Emergency" stations and should work as I imagine. This also means that the Ox Detector needs to be "toggle-able" for High or No signal on detect.

Hopefully this all makes sense

**Comment by radfast** [comment]
I agree we should make the machines consume no power when in an "off" state.

The Oxygen Detector set up, can this be achieved using a redstone torch inverter?

**Comment by TheQuadShot** [comment]
To use an inverter I have to use a repeater then a block. Which is ugly and too big imho. I would like to use the detector to turn on lights when O2 is Present and turn on a bubble gen or flashing light/red light when it is absent.

**Comment by kouwei32** [comment]
One solution to this problem is to use the Atomic Battery in GC 4, which is an infinite battery with low output, basically. You'll have to go to Venus though.

**Comment by TheQuadShot** [comment]
@kouwei32 
Please don't take this wrong, and if you have any questions Please email me directly(go to my profile there's a link). This is not a solution, it does not solve the bug that is in the title. You have provided a workaround for a specific case that I was working on at the time it was filed(see opened date), and is being used as an example/reason why it should/n't be fixed or modified. It also brings in other issues having to do with core functionality that should/n't be dealt with at the same time. The discussion isn't really about working around it, but addresses some of the other issues that may or may not be directly impacted by the fix. This is a bug tracking tool used by the development team, it's not a place to discuss a workaround in one of my builds. The things that should be mentioned here should be additional information about the issue that may help the development team. That being said there are exceptions, however those are at the sole discretion of the development team, and you will find that they are still very narrow in scope or specific to a topic. The dev team has provided us with a place to talk about things like this at the forums located here: https://forum.micdoodle8.com/index.php

We(the galacticraft community) want player input, and involvement. The development team welcomes help here, provided it's reducing the workload they have to deal with. For example, you read a bug report and notice that the person is using block _x_ incorrectly, and no one else had replied yet, then telling them how to correctly use _x_ and providing the link to the wiki and asking them to close the issue would be a great help to the development team. They(the dev team) get tons of email, have real jobs, and family, and yes despite vicious rumours lives. remember that they get an email anytime anything is added/removed/edited in the database, some on mobile devices, so reducing the number they have to open is always appreciated.

**Comment by radfast** [comment]
> and yes despite vicious rumours lives. 

lol, not sure about that one

**Comment by jaz000** [comment]
The problem is it will drain your entire power network (if you have multiple machines connected). Howabout an off button where it dumps the power but won't drain anymore until you turn it back on again?
